config/routes.rb

```diff
@@ -98,7 +98,9 @@ def scoped_navigation_routes(context, navigation)
 
       resources :dependents, only: [:index, :new, :create, :edit, :update, :destroy]
 
-      get "/sign-up", to: redirect('/')
+      resources :signups, only: [:new, :create], path: "sign-up", path_names: { new: '' } do
```

Comment:
What's the significance of this change and the change on the line below?

Reply:
It adds back the gyr signup flow routes instead of redirecting signups to root path.